src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/ImportPage/import.html.twig

```diff
@@ -27,31 +27,33 @@
 {% trans_default_domain 'Admin.Advparameters.Feature' %}
 
 {% block content %}
-    {% if importForm is defined %}
-    <div class="container">
+  {% if importForm is defined %}
```

Comment:
why two spaces here and four spaces above?

Reply:
i cant see 4 spaces above. :/ what do you mean?

Reply:
Sorry speaking about `import_panel.html.twig` file line 34 / 35 6 spaces and line 167 / 168 there is 4 spaces

Reply:
done.